projects/observability/src/shared/dashboard/data/graphql/specifiers/metric-timeseries-specification.model.test.ts

```diff
@@ -1,24 +1,18 @@
 import { IntervalDurationService, isEqualIgnoreFunctions, TimeDuration, TimeUnit } from '@hypertrace/common';
 import { createModelFactory } from '@hypertrace/dashboards/testing';
 import { MetricAggregationType } from '@hypertrace/distributed-tracing';
-import { MODEL_PROPERTY_TYPES } from '@hypertrace/hyperdash-angular';
 import { mockProvider } from '@ngneat/spectator/jest';
 import { ObservabilitySpecificationBuilder } from '../../../../graphql/request/builders/selections/observability-specification-builder';
 import { MetricTimeseriesSpecificationModel } from './metric-timeseries-specification.model';
 
 describe('Metric timeseries specification model', () => {
+  const modelFactory = createModelFactory();
```

Comment:
This test and the similar ones had a pre-existing bug - factories cannot be instantiated inside tests (they set up beforeEach/afterEach).  They had previously provided dummy property types to work around that.